### PR TITLE
Milestone 02: Query update table :fast_forward: 

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### In this branch, I have worked on the following things :+1: 

:heavy_check_mark: Updated Schema (Added Species Column)
:heavy_check_mark: Added Transactions (Updated Species Data)
:heavy_check_mark: Added Queries (Data with Aggregate Functions)
:heavy_check_mark: Following Gitflow
:heavy_check_mark: Documented work in a professional way

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,5 +6,6 @@ name text,
 date_of_birth date,
 escape_attempts int,
 neutered boolean,
-weight_kg decimal
+weight_kg decimal,
+species text
 );
```
Comment (andersonlebon) [inline schema.sql]: - [ ] Kindly note that you were supposed to run a spacial query for adding the species column in the animal table and not to change the query that creates the animal table. Hint :point_down: 

```sql
   ALTER TABLE animals ADD species varchar(255);
```

File: data.sql [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
-INSERT INTO animals VALUES (2,'Gabumon', '2018-NOV-15', 2, TRUE, 8);
-INSERT INTO animals VALUES (3,'Pikachu', '2021-JAN-07', 1, FALSE, 15.04);
-INSERT INTO animals VALUES (4,'Devimon', '2017-MAY-12', 5, TRUE, 11);
+INSERT INTO animals VALUES (1,'Charmender', '2020-Feb-08', 0, false, -11);
+INSERT INTO animals VALUES (2,'Plantom', '2021-Nov-15', 2, true, -5.7);
+INSERT INTO animals VALUES (3,'Squirtle', '1993-Apr-02', 3, false, -12.13);
+INSERT INTO animals VALUES (4,'Angemon', '2005-Jun-12', 1, true, -45);
+INSERT INTO animals VALUES (5,'Boarmon', '2005-Jun-07', 7, true, 20.4);
+INSERT INTO animals VALUES (6,'Blossom', '1998-Oct-13', 3, true, 17);
+INSERT INTO animals VALUES (7,'Ditto', '2022-May-14', 4, true, 22);
```
Comment (andersonlebon) [inline data.sql]: - [ ] Kindly you are not supposed to remove your previous data. This milestone is the continuation of the previous one. Kindly add your previous data.

File: queries.sql [hunk 3]
```diff
@@ -1,10 +1,70 @@
...
 
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE TO_CHAR(date_of_birth, 'yyyy' ) BETWEEN '2016' AND '2019';
-SELECT name FROM animals WHERE neutered = true and escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = true;
-SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+-- TRANSACTION 01
```
Comment (andersonlebon) [inline queries.sql]: - [ ] Kindly note that all your previous milestone queries should be present in this milestone. Kindly add them

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @HAMMAS-SALEEM 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (HAMMAS-SALEEM) [reply schema.sql]: Thanks. I have implemented it.

File: data.sql (hunk 2 above)
Comment (HAMMAS-SALEEM) [reply data.sql]: Thanks. I have implemented it.

File: queries.sql (hunk 3 above)
Comment (HAMMAS-SALEEM) [reply queries.sql]: Thanks. I have implemented it.

**Review by julie-ify — APPROVED** [review]
Hi @HAMMAS-SALEEM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**